service/raft/raft_group0.cc

```diff
@@ -209,6 +209,15 @@ future<> raft_group0::abort() {
     co_await _shutdown_gate.close();
 }
 
+future<> raft_group0::start_existing_group0_server(raft::group_id group0_id) {
```

Comment:
I think the name of this function should be synchronized with create_server_for_group. Something start_server_for_group would be better. And since we're no longer here starting other groups, both names should mention group0. How about create_server_for_group0, start_server_for_group0?

Reply:
While I support having this function I don't think we should have two code paths which start an existing group0 server. But we can discuss this in the later patch.

Reply:
Bottom line: please rename this function to start_server_for_group0, please rename create_server_for_group to create_server_for_gropu0.